pages/lyrics/[id].vue

```diff
@@ -0,0 +1,221 @@
+<script setup lang="ts">
+import { ContributorApi, LyricsApi } from "@bcc-code/bmm-sdk-fetch";
+import type { ContributorModel, Lyrics } from "@bcc-code/bmm-sdk-fetch";
+
+const route = useRoute("lyrics-id");
+
+const lyrics = ref<Lyrics>();
+
+onMounted(async () => {
+  lyrics.value = await new LyricsApi().lyricsIdGet({
+    id: Number(route.params.id),
+  });
+
+  // Default longCopyright if not set
+  lyrics.value.longCopyright ??= `© Stiftelsen Skjulte Skatters Forlag, Norway. All rights reserved.
```

Comment:
setting that value should be done when sending to the server. Otherwise you have to select all text and replace it. Same applies to the verses as well.

Reply:
Ah, I see
So this shouldn't be visible in the UI at all before writing something manually? I guess we could use the `placeholder` attribute on the copyright textfield, at least